### DISCUSSION TITLE
Cannot install from AUR according to instructions

### DISCUSSION THREAD

**Discussion by kozross** [rocm-arch/rocm-arch#946] [Support]
When I try to follow the README instructions, I instead get this:

```
[koz@Sebastian ~]$ paru -S rocm-hip-sdk rocm-opencl-sdk
:: Resolving dependencies...
:: There are 2 providers available for rocm-hip-sdk:
:: Repository AUR:
    1) rocm-hip-sdk  2) opencl-amd-dev  
Enter a number (default=1): 1
:: There are 2 providers available for rocm-opencl-sdk:
:: Repository AUR:
    1) rocm-opencl-sdk  2) opencl-amd-dev  
Enter a number (default=1): 1
error: could not find all required packages:
    hsakmt-roct (wanted by: rocm-hip-sdk -> rocm-hip-libraries -> rocm-hip-runtime -> rocm-language-runtime)
    comgr (wanted by: rocm-hip-sdk -> rocm-hip-libraries -> rocm-hip-runtime -> rocm-language-runtime)
    rocprim (wanted by: rocm-hip-sdk -> hipcub)
```

Am I missing something here?

**Comment by w9n** [comment]
seems like these packages have been moved from AUR to community-testing:
https://archlinux.org/packages/community-testing/x86_64/rocprim/
community-testing needs to be activated in /etc/pacman.conf

**Reply by tpkessler** [reply]
Yes, that's correct. I've been maintaining them now as a Arch Linux package maintainer (trusted user), see https://github.com/rocm-arch/rocm-arch/discussions/937 I deleted the packages here and on the AUR as they are not actively maintained anymore. We're currently testing the ROCm stack on different GPUs and I hope we can move it to [community] soon.
